utils.js

```diff
@@ -38,6 +38,23 @@ function returnRandomObjectPropertiesAndValues(collection, ammount){
 
 }
 
+/**-------------------------------------------------------------------------------
+ * @description Calculates the value by number of dice and dice sides.
+ * Example: diceThrow(2,8) = 2d8 or diceThrow(10) = 1d6
+ * @param {Number} diceQuantity - integer for the quantity of dice to throw
+ * @param {Number} diceSides - integer for the number of sides 2, 4, 6, 8, 10, 12, 20, ...
+ * @returns {Number} integer with the result of dice throw.  
+*///------------------------------------------------------------------------------
+function diceThrow (diceQuantity = 1, diceSides) {
```

Comment:
This is just a default value, actually it would be better if it was

>  function diceThrow (diceSides, diceQuantity = 1 )

this way you could call the function as diceThrow(8) and get a return of 1d8